src/operon-runtime/cli.ts

```diff
@@ -30,7 +30,7 @@ program
     const openapi = await generateOpenApi(entrypoint);
     if (openapi) {
       const filename = path.join(path.dirname(entrypoint), "openapi.yaml");
-      const yaml = `# OpenApi specification generated by Operon\n\n` + YAML.stringify(openapi);
+      const yaml = `# OpenApi specification generated by Operon\n\n` + YAML.stringify(openapi, { aliasDuplicateObjects: false });
```

Comment:
Do we still have duplicate objects if we use components.parameters?

Reply:
YAML's [Anchors and Aliases](https://yaml.org/spec/1.2.2/#3222-anchors-and-aliases) feature allows you to name a YAML block with an `&` prefixed name and then later refer to it with that name prefixed with `*` instead of `&`. 

Frankly, I had never seen it before the `YAML.stringify` call emitted them. I thought it was a bug at first! :rofl:. In this specific case, since the idempotency key parameter object is reused, it triggers the use of aliases. But the YAML object it is aliasing is a `$ref` object that is already an parameter defined in components.parameters. I felt that the intersection between YAML level anchors/aliases and OpenAPI $refs was confusing so I disabled their use in the generated code. 

Additionally, since this is generated YAML instead of hand written, the anchor/alias doesn't have a meaningful name (the YAML library defaults to letters of the alphabet, so we get `$a` and `*a`).

If we feel that anchor/alias support is important, I can see what mechanisms there are to control the anchor/alias name. But that work, if we choose to do it, would be a future PR